src/System.ComponentModel.TypeConverter/tests/Helpers.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I moved the System.Drawing.Common.Tests/Helpers.cs file into the Common path: https://github.com/dotnet/corefx/blob/master/src/Common/tests/System/Drawing/Helpers.cs

Could you instead use that file and now we can delete this duplicate file? :smile: